scripts/lochness_create_template.py

```diff
@@ -382,10 +429,21 @@ def create_example_meta_file_advanced(metadata: str,
         source_col = col_input_to_col_dict[source]
         if source == 'xnat':
             value = f'xnat.{project_name}:subproject:subject01'
+        elif source_col == 'REDCap':
+            if 'REDCap' in df.loc[0]:
+                prev_value = f'{df.loc[0]};'
```

Comment:
This was my fake metadata for previous tests and I see that the columns are `,` separated:

```vim
Active,Consent,Subject ID,REDCap
1,2021-07-13,LA06749,redcap.FAKE_LA:LA06749
1,2021-07-31,LA10782,redcap.FAKE_LA:LA10782
1,2021-05-13,LA24153,redcap.FAKE_LA:LA24153
1,2021-07-09,LA27050,redcap.FAKE_LA:LA27050
1,2021-07-20,LA54257,redcap.FAKE_LA:LA54257
1,2021-07-05,LA99243,redcap.FAKE_LA:LA99243
```

But here you have put a `;`. Are we missing something?

Reply:
by inserting `;` between the `main redcap` and the `UPENN redcap`, lochness will pull from both REDCap repository.